README.md

```
@@ -1,46 +1,138 @@
+2. **Foreground Notifications**: Managed via the app's notification system when active
+3. **Token Registration**: Automatically registers FCM tokens with the backend
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+The service worker is built using esbuild and is automatically included in the build process.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Project Structure
 
-### `yarn eject`
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)